src/agent/Makefile

```diff
@@ -146,7 +146,7 @@ run:
 install-services: $(GENERATED_FILES)
 ifeq ($(INIT),no)
 	@echo "Installing systemd unit files..."
-	$(foreach f,$(UNIT_FILES),$(call INSTALL_FILE,$f,$(UNIT_DIR)))
+	$(foreach f,$(UNIT_FILES),$(call INSTALL_FILE,$f,$(DESTDIR)/$(UNIT_DIR)))
```

Comment:
In src/agent/Makefile:
`DESTDIR :=`
In src/runtime/Makefile:
`DESTDIR ?= /`

I didn't check other Makefiles nor variables but I think we should make the default values consistent across makefiles as well. In a follow up PR?